src/base/_bootstrap-4.scss

```diff
@@ -0,0 +1,34 @@
+/*
+  WET-BOEW
+  @title: Bootstrap 4 suplemental style for WET-BOEW
+ */
+
+/* Padding */
+.p-0 {
+	padding: 0 !important;
+}
+
+.pl-2
+.px-2 {
+	padding-left: 5px !important;
+}
+
+.pt-4,
+.py-4 {
+	padding-top: 30px !important;
```

Comment:
Doesn't the `.py-4` also have a `padding-bottom: 30px !important;`?

Reply:
It originally had `padding-top: 1.5rem !important;` so should be 24px.

Reply:
The 30 px was calculated based on size of mrgn-tp-lg

Reply:
The real `rem` from WET is based on 10px unfortunately, that's the only reason stopping us from using the actual `rem`. Consequently, it is better (at least temporarily) to base our numbers on what we already have with the `mrgn` `-tp` `-bttm` etc.